manager/loginmanager/login.go

```diff
@@ -57,6 +46,16 @@ func SocialLoginUser(userStore *store.UserStore, accessGenerate *generates.JWTAc
 		if err != nil {
 			return nil, err
 		}
+	} else if err == errors.ErrInvalidUser {
+		// If user does not exists
+		createErr := usermanager.CreateSocialUser(userStore, user)
+		if createErr != nil {
+			return nil, createErr
+		}
+		storedUser = user
```

Comment:
Why this assignment?

Reply:
`storedUser` variable is the one from which information is extracted for the token in line no. 63. In above if blocks also `storedUser` is the variable which is being updated so as to extract the updated information for the token.

Reply:
This is later used to generate the jwt token.